articles/clients/index.md

```diff
@@ -46,6 +46,22 @@ A new client will be created and you will be redirected to this client's view th
 
 - [Connections](${manage_url}/#/clients/${account.clientId}/connections): Connections are sources of users. They are categorized into Database, Social and Enterprise and can be shared among different clients. For more details refer to: [Connections](/clients/connections). For a detailed list on the supported Identity Providers refer to: [Identity Providers Supported by Auth0](/identityproviders).
 
+### Assign an Environment Tag
+
+After creating a client, make sure to assign environment tags to your tenants to differentiate between development, staging, and production environments.
```

Comment:
This is not a mandatory step each time you create a new client, so I would rephrase `make sure to assign environment tags to your tenants` to sth like make sure that your tenant has the proper environment set.

Reply:
I would also add a note about setting up multiple environments and how the info can be found here https://auth0.com/docs/dev-lifecycle/setting-up-env